mito-ai/src/components/AgentToolComponents/ErrorFixupToolUI.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { classNames } from '../../utils/classNames';
+import { getContentStringFromMessage } from '../../utils/strings';
+import OpenAI from 'openai';
+import '../../../style/ErrorFixupToolUI.css';
+import ExpandIcon from '../../icons/ExpandIcon';
+import WrenchAndScrewdriverIcon from '../../icons/WrenchAndScrewdriverIcon';
+
+interface IErrorFixupToolUIProps {
+    message: OpenAI.Chat.ChatCompletionMessageParam;
+}
+
+const parsePythonErrorType = (content: string | undefined): string => {
+    if (!content) return 'Error';
+    const errorMatch = content.match(/(\w+Error):/);
+    return errorMatch?.[1] || 'Error';
+};
+
+const ErrorFixupToolUI: React.FC<IErrorFixupToolUIProps> = ({ message }) => {
+    const [isExpanded, setIsExpanded] = useState(false);
+    const messageContent = getContentStringFromMessage(message);
+
+    return (
+        <div className={classNames('error-fixup-container', {
+            'error-fixup-collapsed': !isExpanded
+        })}>
+            <div
+                onClick={() => setIsExpanded(!isExpanded)}
+                className={classNames('error-fixup-toggle', {
+                    expanded: isExpanded
+                })}
+            >
+                <span className="error-fixup-toggle-content">
+                    <WrenchAndScrewdriverIcon />
+                    Fixing {parsePythonErrorType(messageContent)}
+                </span>
+                <ExpandIcon isExpanded={isExpanded} />
+            </div>
+            {isExpanded && (
+                <div className="error-fixup-expanded">
+                    {messageContent}
```

Comment:
The message content contains python code. Currently, the python code not formatted and is displayed in ```

Maybe it would be better to just delete the python code section (we have utility functions, like `splitStringWithCodeBlocks` that should make this pretty easy) and just display the error? 

Or if you think the python code should remain, then let's render it like python code! 

<img width="594" alt="Screenshot 2025-07-01 at 1 09 47 PM" src="https://github.com/user-attachments/assets/ba25d501-bca4-4454-a298-2e1750c4b3b6" />